test/unit/env.js

```diff
@@ -0,0 +1,27 @@
+import jQuery from 'jquery';
+
+global.$ = global.jQuery = jQuery;
+require('bootstrap');
+global.welderApiHost = 'localhost';
+global.welderApiScheme = 'HTTP';
+
+// <mockLocalStorage>
+const localStorageMock = (() => {
```

Comment:
Hmmm, let's leave it for now but I think this file was only used for debugging or something like that. We should investigate if we can remove it and drop the mock.